src/mpid/ch4/shm/ipc/src/ipc_win.c

```diff
@@ -66,7 +66,7 @@ int MPIDI_IPC_mpi_win_create_hook(MPIR_Win * win)
     MPIDIG_win_shared_info_t *shared_table = NULL;
     IPC_win_shared_info_t *ipc_shared_table = NULL;     /* temporary exchange buffer */
     int *ranks_in_shm_grp = NULL;
-    MPIDI_SHM_IPC_type_t ipc_type = MPIDI_SHM_IPC_TYPE__NONE;
+    MPIDI_SHM_IPC_type_t ipc_type = attr.ipc_type;
 
```

Comment:
Why is it set before calling `MPIDI_IPC_get_mem_attr `?

Reply:
just mistake